Stripe/STPPaymentOptionTuple.h

```diff
@@ -20,7 +20,8 @@ NS_ASSUME_NONNULL_BEGIN
 
 + (instancetype)tupleWithPaymentOptions:(NSArray<id<STPPaymentOption>> *)paymentOptions
                   selectedPaymentOption:(nullable id<STPPaymentOption>)selectedPaymentOption
-                      addApplePayOption:(BOOL)applePayEnabled;
+                      addApplePayOption:(BOOL)applePayEnabled
```

Comment:
can we get rid of the apple pay specific param and just use additionalOptions?

Reply:
I did originally, but found that `applePayEnabled` is a little more complex than that. We need the `appleMerchantIdentifier` from STPPaymentConfiguration, which the STPPaymentOptionTuple isn't aware of, and I don't really want to pipe it all the way down here.

Reply:
For reference:

```
- (BOOL)applePayEnabled {
    return self.appleMerchantIdentifier &&
    (self.additionalPaymentOptions & STPPaymentOptionTypeApplePay) &&
    [Stripe deviceSupportsApplePay];
}
```